sourcecode/apis/contentauthor/app/Game.php

```diff
@@ -142,4 +143,14 @@ protected function getTags(): array
             'h5p:' . $this->getMachineName(),
         ];
     }
+
+    public function getMaxScore(): int|null
+    {
+        try {
+            $handler = $this->getGameTypeHandler();
+            return $handler->getMaxScore();
+        } catch (Exception) {
```

Comment:
This will catch:

* Notices/warnings converted to ErrorException
* Database exceptions
* Possible future exceptions from e.g. Laravel we can't know about

... and leave us with no idea any of this happened.

Presumably this is meant to catch 'game type not found' exceptions, so that should just be its own error class, and this statement changed to catch only those instead.